### PR TITLE
feat: Initial UI part for A/V moderation.

### PR DESCRIPTION
**Pull request by damencho**
Based on https://github.com/jitsi/jitsi-meet/pull/7779

Co-authored-by: Gabriel Imre <gabriel.lucaci@8x8.com>

### CODE REVIEW

File: react/features/base/participants/middleware.js [hunk 1]
```diff
@@ -110,6 +111,29 @@ MiddlewareRegistry.register(store => next => action => {
...
+            // XXX Only the local participant is allowed to update without
+            // stating the JitsiConference instance (i.e. participant property
+            // `conference` for a remote participant) because the local
+            // participant is uniquely identified by the very fact that there is
+            // only one local participant.
+
+            id: localId,
+            local: true,
+            raisedHand: enabled
+        }));
```
Comment (gimre) [inline react/features/base/participants/middleware.js:128]: can just use the raiseHand action

Comment (damencho) [reply react/features/base/participants/middleware.js:128]: Thanks

Comment (damencho) [reply react/features/base/participants/middleware.js:128]: This is the raise hand implementation.

Comment (damencho) [reply react/features/base/participants/middleware.js:128]: this is the implementation of the action. The action triggers this middleware handling.

**Comment by vp8x8** [comment]
Jenkins please test this please.

**Comment by vp8x8** [comment]
Created a branch which fixes the failing tests https://github.com/jitsi/jitsi-meet-torture/tree/av-moderation-ui but for some reason is not being picked up 🤷‍♂️

**Comment by damencho** [comment]
You need and a PR from that branch

**Review by vp8x8 — DISMISSED** [review]

**Review by vp8x8 — DISMISSED** [review]

**Review by vp8x8 — DISMISSED** [review]

**Review by vp8x8 — APPROVED** [review]

**Pull request merged**